includes/TypesForLines/LineMeshRefinement.js

```diff
@@ -1,8 +1,163 @@
+* @param	{Number or Array}	lines					Lines assigned to this refinement, can be undefined
+* @param	{String}			comment					Comment, can be undefined
+* @param	{Object}			params					line mesh refinement parameters, can be undefined
+* @return	{Object}			Created line mesh refinement HLF
+*/
 function LineMeshRefinement(no,
+							target_length,
+							layers,
+							lines,
                             comment,
                             params)
 {
-    var line_mesh_refinement = engine.create_line_mesh_refinement(no);
-    set_comment_and_parameters(line_mesh_refinement, comment, params);
-    return line_mesh_refinement;
+    this.settings = createLineMeshRefinement(no, target_length, layers, lines, comment, params);
+    var self = this;
+    return self;
 }
+
+
+/**
+* Change line mesh refinement to type based on element length
+* @param	{Number}			target_length			Length of FEA element side on the line, can be undefined
+* @param	{Number}			layers					Number of layers to be affected by this refinement, can be undefined
+* @param	{Number or Array}	lines					Lines assigned to this refinement, can be undefined
+* @return	{Object}			Created line mesh refinement
+*/
+LineMeshRefinement.prototype.Lenght = function(target_length, layers, lines) {
+	
+	this.settings.type = line_mesh_refinements.TYPE_LENGTH;
+	if (target_length != undefined) {
+		this.settings.target_length = target_length;
+	}
+
+	if (layers != undefined) {
+		this.settings.number_of_layers = layers;
+	}
+
+	if (lines != undefined) {
+		this.settings.lines = lines;
+	}
+
+	return this.settings
+}
+
+
+/**
+* Change line mesh refinement to type based on number of elements on line
+* @param	{Number}			elements			Number of FEA elements on the line, can be undefined
+* @param	{Number}			layers				Number of layers to be affected by this refinement, can be undefined
+* @param	{Number or Array}	lines				Lines assigned to this refinement, can be undefined
+* @return	{Object}			Created line mesh refinement
+*/
+LineMeshRefinement.prototype.Elements = function(elements, layers, lines) {
+
+	this.settings.type = line_mesh_refinements.TYPE_ELEMENTS;
+	if (elements != undefined) {
+		this.settings.elements_finite_elements = elements;
+	}
+
+	if (layers != undefined) {
+		this.settings.number_of_layers = layers;
+	}
+
+	if (lines != undefined) {
+		this.settings.lines = lines;
+	}
+	return this.settings
+}
+
+
+/**
+* Change line mesh refinement to type gradually changed due to distance from the line
+* @param	{Number}			gradual_rows			Number gradual rows of FEA elements, can be undefined
+* @param	{Number}			layers					Number of layers to be affected by this refinement, can be undefined
+* @param	{Number or Array}	lines					Lines assigned to this refinement, can be undefined
+* @return	{Object}			Created line mesh refinement
+*/
+LineMeshRefinement.prototype.Gradual = function(gradual_rows, layers, lines) {
```

Comment:
USe name closer to RFEM - Gradually
no, name, lines, numberOfRows, numberOfLayers